db/migrate/20230316193324_add_confirmation_fields_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddConfirmationFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string
+    add_index :users, :unconfirmed_email
```

Comment:
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/65068771/225776445-c37226ff-e791-49d8-84f9-a2f12d668edb.png">

- [ ] Kindly note that all of these columns you are trying to add exist already in your user table that's why I am getting the error above when trying to run your migrations

Reply:
Dear @andersonlebon 

Thank you for pointing that out. i was having a problem from my side, so to solve it, i had put that new migration. I will remove it and re-submit since on your side all is okay.